eng/common/pipelines/templates/steps/generate-sdk.yml

```diff
@@ -0,0 +1,172 @@
+parameters:
+  - name: SpecRepoOwner
+    type: string
+  - name: SpecRepoName
+    type: string
+  - name: SdkRepoOwner
+    type: string
+  - name: SdkRepoName
+    type: string
+  - name: IntegrationSdkRepoOwner
+    type: string
+    default: ''
+  - name: IntegrationSdkRepoName
+    type: string
+    default: ''
+  - name: SpecRepoCommit
```

Comment:
nit: could we sort these so all spec properties are together and then the SDKs ones?

Reply:
Created a new version of the pipeline based on the discussion with Wes.